src/components/FormElement.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {ForwardedRef, forwardRef, LegacyRef} from 'react';
+import {View, ViewProps} from 'react-native';
+import * as ComponentUtils from '@libs/ComponentUtils';
+
+type FormElementProps = ViewProps;
+function FormElement(props: FormElementProps, ref: ForwardedRef<View | HTMLFormElement | null>) {
+    return (
+        <View
+            role={ComponentUtils.ACCESSIBILITY_ROLE_FORM}
+            ref={ref as LegacyRef<View>}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+        />
+    );
+}
+
+FormElement.displayName = 'BaseForm';
```

Comment:
is it a proper displayName? dosen't follow file name

Reply:
It was here before so I am not sure if this should be changed in TS migration

Reply:
If you take a look at blame [here](https://github.com/Expensify/App/pull/8730/files#diff-18e707a3ff1640f8eb511c120602ef20afb815985e8329db4e2170732e087cfd) they must've missed it. 

Let's change it to 'FormElement' to follow the guidelines, other than that LGTM!